### ISSUE TITLE
Enable TSC feature for Pentium+ cpus

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vgmurphy/limbo-android#24]
```
Enable TSC feature for Pentium+ cpus as an option for ICS and above devices only
```

Original issue reported on code.google.com by `ideal2i...@gmail.com` on 14 Jan 2013 at 1:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `ideal2i...@gmail.com` on 14 Jan 2013 at 1:27
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Included in 0.9.7
```

Original comment by `ideal2i...@gmail.com` on 5 May 2013 at 9:53
- Changed state: **Done**
